gensim/models/phrases.py

```diff
@@ -209,12 +209,12 @@ def load(cls, *args, **kwargs):
         model = super(PhrasesTransformation, cls).load(*args, **kwargs)
         # update older models
         # if value in phrasegrams dict is a tuple, load only the scores.
-        try:
-            for components, scores in model.__dict__['phrasegrams'].items():
-                if isinstance(scores, tuple):
-                    model.__dict__['phrasegrams'][components] = scores[1]
-        except KeyError:
-            pass
+        if model.phrasegrams:
+            components = model.phrasegrams.keys()
+            for component in components:
```

Comment:
These two lines are better merged into one (so the temporary variable is released as soon as it's not needed).